starboard/nplb/compiler_compliance/cpp_compiler_version_check.cc

```diff
@@ -14,15 +14,18 @@
 
 #include <iostream>
 
+#define COBALT_CLANG_ERROR_MSG "We compile above Starboard targets like Cobalt, Nplb with " \
+    "clang17 or higher. If you are building with a non-Cobalt " \
+    "provided toolchain you may run into this error. To fix this error" \
+    "build libnplb using the Evergreen toolchain : " \
+    "cobalt.dev/development/setup-raspi"
```

Comment:
This shouldn't link to the old developer site, and why is it linking to raspi?

Shouldn't this link to https://developers.google.com/youtube/cobalt/docs/development/setup-linux? or possibly https://developers.google.com/youtube/cobalt/docs/development/setup-raspi if you specifically want raspi.

Reply:
I actually recommended using short link, it's much cleaner to parse in a compiler output. The devsite url is not going anywhere, it's okay to use it as a doc link shortener. 

I'm not sure if raspi or linux is better in this context, i'm not sure it matters much.